### PR TITLE
super basic tracking implementation with google analytics / gtag

### PR DESCRIPTION
**Pull request by UnsungHero97**

### CODE REVIEW

File: src/containers/app/AppHeaderContainer.js [hunk 1]
```diff
@@ -269,6 +278,6 @@ function mapDispatchToProps(dispatch :Function) :Object {
   };
 }
 
-export default withRouter(
+export default withRouter<any>(
```
Comment (solomontang) [inline src/containers/app/AppHeaderContainer.js]: Should probably just make this `<*>` like the others.

File: src/core/tracking/google/GoogleAnalytics.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+/*
+ * @flow
+ */
+
+const GOOGLE_TRACKING_ID :string = 'UA-118446829-3';
```
Comment (solomontang) [inline src/core/tracking/google/GoogleAnalytics.js]: Is this safe to check-in? This isn't like a secret or anything right?

Comment (UnsungHero97) [reply src/core/tracking/google/GoogleAnalytics.js]: Correct. It's not a secret.

https://developers.google.com/analytics/devguides/collection/gtagjs/

File: src/components/buttons/DropdownButtonWrapper.js [hunk 3]
```diff
@@ -1,7 +1,3 @@
-/*
- * @flow
```
[the remark this answers is not in the document]
Comment (UnsungHero97) [reply src/components/buttons/DropdownButtonWrapper.js]: I got my branches mixed up. Good catch.

File: src/containers/app/AppHeaderContainer.js (hunk 1 above)
Comment (UnsungHero97) [reply src/containers/app/AppHeaderContainer.js]: Missed this one, thanks. By the way, `*` is apparently deprecated.

https://github.com/facebook/flow/issues/6308

Comment (UnsungHero97) [reply src/containers/app/AppHeaderContainer.js]: Missed this one, thanks. By the way, `*` is apparently deprecated.

**Pull request merged**